client/src/client.rs

```diff
@@ -452,22 +451,23 @@ impl Client {
         &self,
         transaction: SignedTransaction,
     ) -> Result<HashOf<VersionedSignedTransaction>> {
-        let client = self.clone();
-        let (init_sender, init_receiver) = mpsc::channel();
+        let (init_sender, init_receiver) = tokio::sync::oneshot::channel();
         let hash = transaction.hash();
 
         // TODO: use `std::thread::scope()` after update to Rust 1.63
```

Comment:
How about introducing this change here, as long as we're already touching that bit of code? Or rather, how much do we need to actually switch it to `std`?

Reply:
Sure, i will introduce this change.

> Or rather, how much do we need to actually switch it to std?

We will be able to remove crossbeam dependency from client.